Error: Cannot destroy VoiceConnection - it has already been destroyed

```
Error: Cannot destroy VoiceConnection - it has already been destroyed
    at G.destroy (vutd-shovel\node_modules\@discordjs\voice\src\VoiceConnection.ts:534:10)   
    at Room.destroy (vutd-shovel\src\classes\room.ts:195:22)
    at Room.<anonymous> (vutd-shovel\src\classes\room.ts:100:14)
    at Generator.next (<anonymous>)
    at fulfilled (vutd-shovel\src\classes\room.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[ERROR] 20:43:01 Error: Cannot destroy VoiceConnection - it has already been destroyed
```

--- a/src/classes/room.ts
+++ b/src/classes/room.ts
@@ -192,7 +192,9 @@ export default class Room {
    * disconnects from voice channel and stop collecting messages.
    */
   destroy() {
-    this.#connection.destroy();
+    if (this.#connection.state.status !== VoiceConnectionStatus.Destroyed) {
+      this.#connection.destroy();
+    }
     this.#messageCollector.stop();
   }
 }

